libs/domain/product/src/mixins/product.mixin.ts

```diff
@@ -15,6 +15,7 @@ export declare class ProductMixinInterface
 {
   sku?: string;
   product?: Product | null;
+  productS: Signal<Product | null>;
```

Comment:
Don't think everyone understand that `S` is a `Signal`. So suggest to use full word.

Reply:
yes, also changed it aleady :)